x12-edi/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,593 @@
+= X12 EDI Connector - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium] +
+X12 EDI Connector Version 2.2.0
+
+X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.
+The schema definitions for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 standard transaction sets] are provided.
+Many HIPAA document versions are also supported, as listed separately in the https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[Connector HIPAA reference page].
+
+== Prerequisites
+
+This document assumes that you are familiar with X12, Mule, Anypoint
+Connectors, Anypoint Studio, Mule flows, and Mule Global Elements.
+
+See the Release Notes for compatibility information. A link to the
```

Comment:
It's not necessary to mention the link in the See Also since the link for the release notes is in the info box on the page, right at the top.